api/models/user.js

```diff
@@ -22,17 +22,11 @@ module.exports = (sequelize, DataTypes) => {
         autoIncrement: true,
         primaryKey: true,
       },
+      username: { type: DataTypes.STRING, unique: true, allowNull: false },
```

Comment:
move this to `userLocals` since, only internal members will have the username and students will not.